### PR TITLE
PyUnusedCodeBear.py: Change docstring

### PR DESCRIPTION
**Pull request by mridubhatnagar**
Closes https://github.com/coala/coala-bears/issues/1832

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: bears/python/PyUnusedCodeBear.py [hunk 1]
```diff
@@ -22,11 +22,12 @@ def run(self, filename, file,
 
         - Unneeded pass statements.
         - Unneeded builtin imports.
+        - Unneeded variables.
 
         :param remove_all_unused_imports:
             True removes all unused imports - might have side effects
         :param remove_unused_variables:
-            True removes unused variables - might have side effects
+            False removes unused variables - might have side effects
```
Comment (userzimmermann) [inline bears/python/PyUnusedCodeBear.py]: `False` **doesn't** remove unused variables! ;) And maybe exchange *unused* with *the unneeded*, to get some consistency with the list above

And you should move `might have side effects` to the `- Unneeded variables.` list entry above

**Comment by mridubhatnagar** [comment]
Hey, it seems based on your recent comment on #1829 . Unneeded needs to be changed to unused. Which is contrary to your comment on #1844. Hence, based on recent comment. Again making changes. I hope I have interpreted it correct.

**Comment by userzimmermann** [comment]
ack https://github.com/coala/coala-bears/pull/1844/commits/998a9166f17351b4cb7e4def1ebbead71e07783e

**Comment by userzimmermann** [comment]
@rultor merge

**Comment by userzimmermann** [comment]
reack https://github.com/coala/coala-bears/pull/1844/commits/8317b5ad8b43a51896f89f01f1b266a11e3e1363

**Comment by userzimmermann** [comment]
@rultor merge

**Comment by userzimmermann** [comment]
@rultor merge

**Comment by userzimmermann** [comment]
@rultor merge

**Comment by userzimmermann** [comment]
@rultor merge

**Comment by userzimmermann** [comment]
reack https://github.com/coala/coala-bears/pull/1844/commits/5772def150e145bee5c49a53153aad48bc609ef0

**Comment by userzimmermann** [comment]
@rultor merge

File: bears/python/PyUnusedCodeBear.py [hunk 2]
```diff
@@ -22,11 +22,12 @@ def run(self, filename, file,
 
         - Unneeded pass statements.
         - Unneeded builtin imports.
+        - Unused variables - might have side effects
 
         :param remove_all_unused_imports:
             True removes all unused imports - might have side effects
         :param remove_unused_variables:
-            True removes unused variables - might have side effects
+            False doesn't remove unused variables
```
Comment (jayvdb) [inline bears/python/PyUnusedCodeBear.py]: this is a double negative.  please dont do that in docstrings.

Comment (userzimmermann) [inline bears/python/PyUnusedCodeBear.py]: @jayvdb Give a better suggestion please ;)

**Comment by jayvdb** [comment]
unack 5772def

File: bears/python/PyUnusedCodeBear.py (hunk 2 above)
Comment (mridubhatnagar) [inline bears/python/PyUnusedCodeBear.py]: Should this be changed to 
False keeps unused variables.

**Comment by userzimmermann** [comment]
ack https://github.com/coala/coala-bears/pull/1844/commits/14b2d4899cdd9b2bcd84fc3aa135b631be4b8f1e

**Comment by userzimmermann** [comment]
@rultor merge

**Pull request merged**